lib/consensus/isaac.go

```diff
@@ -97,31 +97,51 @@ func (is *ISAAC) SelectProposer(blockHeight uint64, round uint64) string {
 //   string = A string explaining why it's not the latest voting basis (when `bool` == `false),
 //   or empty string ("") if the voting basis is valid (when `bool` == `true`).
```

Comment:
If you are changing the signature, don't forget the documentation

Reply:
Oh, sorry I thought that I modified it.